src/containers/email/index.tsx

```diff
@@ -1,41 +1,101 @@
+  EmailerFilters,
+  FilteredSite,
+  FilterSitesParams,
+} from './types';
 import EmailerFilterControls from '../../components/emailerFilterControls';
+import AdoptedSitesTable from '../../components/adoptedSitesTable';
+import protectedApiClient from '../../api/protectedApiClient';
+import { NEIGHBORHOOD_OPTS, Neighborhoods } from '../../assets/content';
 import SendEmailForm from '../../components/forms/sendEmailForm';
-// import protectedApiClient from '../../api/protectedApiClient';
 
 const EmailPageContainer = styled.div`
   width: 90vw;
   margin: 30px auto auto;
 `;
 
-const selectStyles = { marginTop: 20, minWidth: 150 };
+const FilterHeader = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin-top: 15px;
+`;
+
+const FetchInfoContainer = styled.div`
+  text-align: center;
+  padding: 30px;
+`;
+
+const selectStyles = { marginTop: 20, minWidth: 150, marginBottom: 20 };
+
+const defaultFilters: EmailerFilters = {
+  activityCountMin: 0,
+  neighborhoods: [],
+  commonNames: [],
+};
+
+enum LoadingState {
+  SUCCESS = 'success',
+  LOADING = 'loading',
+  ERROR = 'error',
+}
+
+function neighborhoodToId(neighborhood: Neighborhoods): number {
+  return (
+    NEIGHBORHOOD_OPTS.find((opt) => opt.label === neighborhood)?.value ?? -1
+  );
+}
 
 const Email: React.FC = () => {
   const [emailType, setEmailType] = useState<EmailType>(EmailType.INACTIVE);
-  const [filters, setFilters] = useState<EmailerFilters>({
-    activityCountMin: 0,
-    neighborhoods: [],
-    commonNames: [],
-  });
-
-  const selectedEmails = [''];
+  const [filters, setFilters] = useState<EmailerFilters>(defaultFilters);
+  const [fetchData, setFetchData] = useState<FilteredSite[]>([]);
+  const [fetchSitesState, setFetchSitesState] = useState<LoadingState>(
+    LoadingState.SUCCESS,
+  );
+  const [selectedEmails, setSelectedEmails] = useState<string[]>([]);
 
-  // const treeCommonNames = ['Northern red oak', 'Zelkova'];
-  // const neighborhoodIds = [34];
+  function onClickSearch() {
+    setFetchSitesState(LoadingState.LOADING);
+    const req: FilterSitesParams = {
+      treeCommonNames:
+        filters.commonNames.length > 0 ? filters.commonNames : null,
+      adoptedStart: filters.adoptedStart ?? null,
+      adoptedEnd: filters.adoptedEnd ?? null,
+      lastActivityStart: filters.lastActivityStart ?? null,
+      lastActivityEnd: filters.lastActivityEnd ?? null,
+      neighborhoodIds:
+        filters.neighborhoods.length > 0
+          ? filters.neighborhoods.map(neighborhoodToId)
+          : null,
+    };
 
-  // const handleFilterSites = () => {
-  //   return protectedApiClient
-  //     .filterSites(null, null, null, null, null, neighborhoodIds)
-  //     .then((res: FilterSitesResponse) => console.log(res.filteredSites));
-  // };
+    protectedApiClient
+      .filterSites(req)
+      .then((res) => {
+        setFetchSitesState(LoadingState.SUCCESS);
+        setFetchData(
+          res.filteredSites.filter(
```

Comment:
❓Are the min/max activity count filters not applied to the filtered sites that are returned from the route? If so, I think we should look into this in the backend

Reply:
Yeah, I think it makes more sense to add the activity counts as a parameter in the backend route.